test/gtest/bn_fwd_train.cpp

```diff
@@ -144,7 +144,7 @@ INSTANTIATE_TEST_SUITE_P(Smoke,
 
 INSTANTIATE_TEST_SUITE_P(Smoke,
                          GPU_BN_FWD_Train_Large_FP64,
-                         testing::Combine(testing::ValuesIn(NetworkSmall<BNTestCase>()),
+                         testing::Combine(testing::ValuesIn(NetworkLarge<BNTestCase>()),
```

Comment:
Shall it be a new `Full` instance? Let's keep `Smoke` tests relatively small, that's the point.

Reply:
Hi @CAHEK7  could you clarify what you mean by "Full" instance? For each data type, there's 4 sets of test data for NetworkSmall and 32 for NetworkLarge.

Reply:
This is not a new Full instance. Its just there was a typo before. It should be calling NetworkLarge for testing AP2. For AP1 we only test small bn network (4 test).